README.md

```diff
@@ -1,4 +1,4 @@
-﻿<!-- markdownlint-disable-next-line -->
```

Comment:
previously did you remove this line?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable